pkg/client/client_v2.go

```diff
@@ -174,6 +174,9 @@ type V2 interface {
 	//
 	// nil can be returned when the client has never connected.
 	AgentInfo() *AgentInfo
+	// PackageVersion returns the version of the agent package that should be
+	// presented in user visible fields and messages.
+	PackageVersion() string
```

Comment:
I was thinking instead of having this method, the agent could send the whole AgentInfo together with the ConnInfo, that way the client could already populate its local AgentInfo and instead of a new method, the agent info could be used since the very beginning. Without having to wait the 1st check-in.

Reply:
I prefer that. Have it send the entire AgentInfo in the ConnInfo.